### PR TITLE
Admin section links for help and public store link is opened in new tab

### PR DESCRIPTION
**Pull request by manojkulkarni30**
#2022  A new field called OpenUrlInNewTab is added in SiteMapNode.cs file which allows to configure particular url to open in new tab through sitemap.config file. @AndreiMaz please review.

### CODE REVIEW

File: src/Presentation/Nop.Web/Administration/Views/Shared/Menu.cshtml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     if (!String.IsNullOrEmpty(url))
     {
         <li @(isActive ? Html.Raw("class=\"active current-active-item\"") : null)>
-            <a href="@url" class="menu-item-link">
+            <a href="@url" class="menu-item-link" @(item.OpenUrlInNewTab ? Html.Raw("target=\"_blank\"") : null)>
```
Comment (AndreiMaz) [inline src/Presentation/Nop.Web/Administration/Views/Shared/Menu.cshtml]: It's better to remove space before @. And add it inside "Html.Raw". This way no empty space is generated with "OpenUrlInNewTab" set to "false"

File: src/Presentation/Nop.Web/Administration/Views/Shared/_AdminLayout.cshtml [hunk 2]
```diff
@@ -142,7 +142,7 @@
                                     @Html.Widget("admin_header_middle", area: "Admin")
                                 </li>
                                 <li>
-                                    <a href="@Url.Action("Index", "Home", new {area = ""})">
+                                    <a href="@Url.Action("Index", "Home", new {area = ""})" target="_blank">
```
Comment (AndreiMaz) [inline src/Presentation/Nop.Web/Administration/Views/Shared/_AdminLayout.cshtml]: please remove this change. we don't plan to have it

**Comment by manojkulkarni30** [comment]
Thank you @AndreiMaz  for reviewing the PR. I will removed the space before @ in Menu.cshtml, remove the change in _AdminLayout.cshtml and will update the PR.

**Comment by AndreiMaz** [comment]
Thanks a lot! please notify me once it's done

**Comment by AndreiMaz** [comment]
Thanks a lot! please notify me once it's done

**Comment by manojkulkarni30** [comment]
@AndreiMaz updated the pull request. Please review

File: src/Presentation/Nop.Web.Framework/Menu/XmlSiteMap.cs [hunk 3]
```diff
@@ -110,6 +110,18 @@ private static void PopulateNode(SiteMapNode siteMapNode, XmlNode xmlNode)
...
+            // Open URL in new tab
+            var openUrlInNewTabValue = GetStringValueFromAttribute(xmlNode, "OpenUrlInNewTab");
+            bool booleanResult;
+            if (!string.IsNullOrWhiteSpace(openUrlInNewTabValue) && bool.TryParse(openUrlInNewTabValue, out booleanResult))
+            {
+                siteMapNode.OpenUrlInNewTab = booleanResult;
+            }
+            else
+            {
+                siteMapNode.OpenUrlInNewTab = false;
```
Comment (AndreiMaz) [inline src/Presentation/Nop.Web.Framework/Menu/XmlSiteMap.cs]: One more change. I think setting it to "false" is redundant (default value)

**Comment by manojkulkarni30** [comment]
Thank you @AndreiMaz for reviewing the PR. Updated the PR with change. Please review.

**Comment by AndreiMaz** [comment]
Hi Manoj,

Thanks a lot! One question. Could you please please create a new pull request with one single commit? Please also include task number to its comment with "#2022 "

**Pull request closed without merging**